package.json

```diff
@@ -30,6 +30,11 @@
     "turbo": "1.12.5",
     "vitest": "1.4.0"
   },
+  "peerDependencies": {
+    "fuels": ">=0.77.0",
+    "socket.io-client": "4.7.2",
+    "axios": "1.5.1"
+  },
```

Comment:
We should not add dependencies to the root pacakge.json only package specficic. Did you have problem with the current setup? Maybe we ned to turn the package public if this was a problem.